dist/index.html

```diff
@@ -37,6 +38,68 @@ <h1>
+            <h3>About Us</h3>
+            <p> TOP movies is a dynamic film exhibition site with a style that incorporates multifarious features, including state-of-the-art cinema technologies and luxurious dine-in cinema services; administered by a professional operations management.<br>
+                <br>We are driven by the need to continue discovering new and innovative ways of creating inspiring experiences, delivering world-class service, and bringing the magic of cinema to life. Our philosophy is defined by quality, offering luxury and the most premium service affordably, primarily to ensure you receive absolute value. We focus on offering the best service, leaving you that magical experience, every time.
+            </p>
+        </div>
+        <div class="links">
+            <div class="footer-links">
+                <U>Quick Links</U> <br>
+                <div class="bottom-nav">
+                    <a href="#">Home</a>
+                    <a href="#" id="tvshows">Tv Shows</a>
+                </div>
+            </div>
+            <div class="social-links">
+                <u>Connect with us</u> <br>
+                <div class="social-pages">
+                    <a href="#" class="fa fa-facebook"></a>
+                    <a href="#" class="fa fa-twitter"></a>
+                    <a href="#" class="fa fa-instagram"></a>
+                    <a href="#" class="fa fa-telegram"></a>
+                    <a href="#" class="fa fa-youtube"></a>
+                    <a href="#" class="fa fa-linkedin"></a>
+                </div>          
+            </div>
+        </div>
+
+        <div class="footer-section" id="partners">
+            <h3>Partners</h3>
+            <div class="partner-names">
+                <div class="partner">
+                    <img src="./Assets/kalutu.jpeg" alt=" Mr George Ali Partner">
+                    <p> GEORGE ALI</p>
+                </div>
+                <div class="partner">
+                    <img src="./Assets/Tjagu.jpeg" alt="mr Tenerat Jugu partner">
+                    <p> TENERAT JUGU</p>
+                </div>
+                <div class="partner">
+                    <img src="./Assets/Nathaniel.jpeg" alt=" Mr joshua Nathaniel Partner">
+                    <p> JOSHUA NATHANIEL.</p>
+                </div>
+            </div>
+        </div>
+        <div class="bottom-footer">
+
+            <div class="footer-section" id="contact-us">
+                <u>contact-us</u>
+                <p>Email: <a href="#">feedback@TOPmovies.com</a><br>
+                    Phone: +1234567889
+                </p>
+            </div>
+            <div class="footer-legal">
+              <p>&copy; TOPmovies 2023. All rights reserved.</p>
+              <p><a href="#">Privacy Policy</a> | <a href="#">Terms and conditions </a></p>
+              <p> This is a capstone project for cohort 2 of the TOP Developers Institute.</p>
+            </div>
+        </div>
+
```

Comment:
It is best practice not to leave more than one line as whitespace, just leave a maximum of a single line in between elements if you want to space them